ONSdigital/dp-frontend-renderer
# Feature/cookies banner

### What

- Added cookies banner partial - a form that includes an 'Accept all' action and a link to the cookies preferences page
- Added tags for localisation of cookies banner copy
- Added two conditionals to discern whether or not the cookies banner should be rendered - feature flag `EnableCookiesControl` and property `CookiesPreferencesSet`

### How to review

- Check that localisation tags are clear and make sense
- Check that localisation copy aligns with prototype
- Check that markup aligns with requirements

**To test in the browser**
1a. Pull `dp-frontend-dataset-controller` and checkout `feature/flag-cookie-controller`
1b. Pull `sixteens` and checkout `feature/cookies-banner-styling` and run
2. Run the dataset controller. Do so with the flag, `ENABLE_COOKIES_CONTROL=true`, to have the cookies banner render
3. View cookies banner on pages served by the renderer - e.g. `/datasets/cpih01/editions/time-series/versions`
4. Testing the 'Accept all' button would need the `dp-frontend-cookies-controller` running, specifically on the `feature/populate-controller` branch. The route should be `/cookie/accept-all`
5. Set Cookies Preferences button should route to the `/cookies` page (currently pending)

### Who can review

Anyone bar me
